test/__snapshots__/mathml-spec.js.snap

```diff
@@ -185,28 +185,8 @@ exports[`A MathML builder ligatures render properly 1`] = `
         −
       </mo>
       <mtext mathvariant="monospace">
-        \`\`‘Hi----&#x27;&#x27;’
+        \`\`‘Hi----&#x27;&#x27;’\`\`‘Hi----&#x27;&#x27;’
```

Comment:
Why did this snapshot change so much?

Reply:
I think this is because everything has the same `mathvariant` set so our concatenation code is merging all of the `mtext`s into a single `mtext`.